Index: static/templates/chat/opposed.hbs
===================================================================
--- static/templates/chat/opposed.hbs	en
+++ static/templates/chat/opposed.hbs	pl
@@ -1,7 +1,7 @@
 <div class="opposed">
 
     <div class="opposed-message">
-        <p>{{attacker.name}} is targeting {{defender.name}}</p>
+        <p>{{attacker.name}} celuje w {{defender.name}}</p>
     </div>
 
     <div class="opposed-details {{#if result}}{{result.winner}}-winner{{/if}}">
@@ -52,7 +52,7 @@
         {{#if (eq result.winner "attacker")}}
 
             <div class="applied-details">
-            <span> <a class="damage-breakdown"  data-tooltip="{{result.tooltips.damage}}" data-tooltip-direction="LEFT"><i class="fa-regular fa-circle-info"></i></a> {{localize "IMPMAL.DamageDisplay" damage=result.damage}} {{#if applied}} (Applied){{/if}}</span>
+            <span> <a class="damage-breakdown"  data-tooltip="{{result.tooltips.damage}}" data-tooltip-direction="LEFT"><i class="fa-regular fa-circle-info"></i></a> {{localize "IMPMAL.DamageDisplay" damage=result.damage}} {{#if applied}} (Zastosowane){{/if}}</span>
             {{#if applied.rend}}
             <span class="rend">{{localize "IMPMAL.Rend"}} {{applied.rend}}</span>
             {{/if}}
