Index: static/templates/chat/reward.hbs
===================================================================
--- static/templates/chat/reward.hbs	en
+++ static/templates/chat/reward.hbs	pl
@@ -4,7 +4,7 @@
     <div class="rewards">
         {{#if solars}}
         <div>
-            <p>Solars</p>
+            <p>{{localize "IMPMAL.Solars"}}</p>
             <div class="amount">
                 {{solars}}
             </div>
@@ -12,7 +12,7 @@
         {{/if}}
         {{#if xp}}
         <div>
-            <p>XP</p>
+            <p>{{localize "IMPMAL.XP"}}</p>
             <div class="amount">
                 {{xp}}
             </div>
