Index: scripts/DmgP4MH00PsHBHy4.js
===================================================================
--- scripts/DmgP4MH00PsHBHy4.js	en
+++ scripts/DmgP4MH00PsHBHy4.js	pl
@@ -1,5 +1,5 @@
         if (this.actor.system.fate.value > 0)
         {
-            this.scriptNotification("Cannot gain Fate Points", "error");
+            this.scriptNotification("Nie można otrzymać Punktów Przeznaczenia", "error");
             this.actor.update({"system.fate.value" : 0});
         }
\ No newline at end of file
