Index: scripts/GUbhc29q4W2imAXF.js
===================================================================
--- scripts/GUbhc29q4W2imAXF.js	en
+++ scripts/GUbhc29q4W2imAXF.js	pl
@@ -1,7 +1,7 @@
         let psyker = await game.impmal.utility.findId("3GLGsYiH5LiKgYjE");
         let powers = (await game.impmal.utility.getAllItems("power")).filter(i => i.system.discipline == "minor");
         let power = powers[Math.floor(CONFIG.Dice.randomUniform() * powers.length)]?.toObject();
-        this.script.scriptNotification(`Gained ${power.name}`);
+        this.script.scriptNotification(`Otrzymano ${power.name}`);
         psyker = psyker.toObject();
         psyker.effects = [];
         this.actor.createEmbeddedDocuments("Item", [psyker, power]);
\ No newline at end of file
