Index: scripts/i6Hodo3VSdLPvGRn.js
===================================================================
--- scripts/i6Hodo3VSdLPvGRn.js	en
+++ scripts/i6Hodo3VSdLPvGRn.js	pl
@@ -1,1 +1,1 @@
-this.actor.applyDamage(3, {ignoreAP : true}).then(data => this.script.scriptMessage("Took " + data.woundsGained + " Damage"));
\ No newline at end of file
+this.actor.applyDamage(3, {ignoreAP : true}).then(data => this.script.scriptMessage("Otrzymano " + data.woundsGained + " Obrażeń"));
\ No newline at end of file
