Index: scripts/nvmJmxaifzfV9UfG.js
===================================================================
--- scripts/nvmJmxaifzfV9UfG.js	en
+++ scripts/nvmJmxaifzfV9UfG.js	pl
@@ -1,3 +1,3 @@
 let damageData = await this.actor.applyDamage(3 + this.effect.sourceActor.system.characteristics.wil.bonus)
 
-this.script.scriptMessage(`Takes ${damageData.damage} Damage ${damageData.woundsGained != damageData.damage ? `(${damageData.woundsGained} Wounds)` : ""}`);
\ No newline at end of file
+this.script.scriptMessage(`Otrzymano ${damageData.damage} Obrażeń ${damageData.woundsGained != damageData.damage ? `(${damageData.woundsGained} Ran)` : ""}`);
\ No newline at end of file
