Index: scripts/yhhzFRtF3TNjZAIt.js
===================================================================
--- scripts/yhhzFRtF3TNjZAIt.js	en
+++ scripts/yhhzFRtF3TNjZAIt.js	pl
@@ -1,6 +1,6 @@
     let weapons = await Promise.all(["2XMWybp4AQqoEPtI", "blRx8XLXQaD9sjDz", "BoWOcsxxSvrGIefL", "ix1RLHT1T5rd0Egu", "hlcxzIxPsc6fXOhJ", "HQ4thlBy0uFtXo13"].map(game.impmal.utility.findId));
 
-    let choice = (await ItemDialog.create(weapons, 1, {text : "Choose a weapon for the specialist to use"})).map(i => i.toObject())
+    let choice = (await ItemDialog.create(weapons, 1, {text : "Wybierz Broń dla specjalisty do użycia"})).map(i => i.toObject())
 
     if (choice && choice.length)
     {
