Index: static/templates/actor/tabs/actor-combat.hbs
===================================================================
--- static/templates/actor/tabs/actor-combat.hbs	en
+++ static/templates/actor/tabs/actor-combat.hbs	pl
@@ -273,7 +273,7 @@
 
         <div class="flexrow" style="align-items: start">
             <div class="hit-locations sheet-list">
-                <div class="list-header">Protection</div>
+                <div class="list-header">Ochrona</div>
                 <div class="list-content">
                 {{#each hitLocations}}
                     <div class="location list-row no-scan" data-key="{{this.key}}">
