Index: static/templates/partials/defending-against.hbs
===================================================================
--- static/templates/partials/defending-against.hbs	en
+++ static/templates/partials/defending-against.hbs	pl
@@ -3,7 +3,7 @@
     <div class="attack-details">
         <label>{{localize "IMPMAL.DefendingAgainst"}}</label>
         <div class="gradient-border" style="width: 100%"></div>
-        <div><span>{{tokenName data.actor}}</span><span>{{data.context.title}}</span><span>{{data.result.SL}} SL</span></div>
+        <div><span>{{tokenName data.actor}}</span><span>{{data.context.title}}</span><span>{{data.result.SL}} {{localize "IMPMAL.SL"}}</span></div>
     </div>
     <a class="remove-opposed" data-action="removeOpposed"><i class="fa-solid fa-xmark"></i></a>
 </div>
\ No newline at end of file
